DataFormats/TrackReco/interface/TrackExtraBase.h

```diff
@@ -73,12 +78,15 @@ class TrackExtraBase
 
     }
 
+    TrajParams const & trajParams() const  {return m_trajParams;}

```

Comment:
actually we can even store it... 
It is as useful as most of the rest of the RECO content...
Let's ear the opinion of those concerned